book/src/configuration.md

```diff
@@ -56,6 +56,7 @@ on unix operating systems.
 | `rulers` | List of column positions at which to display the rulers. Can be overridden by language specific `rulers` in `languages.toml` file. | `[]` |
 | `bufferline` | Renders a line at the top of the editor displaying open buffers. Can be `always`, `never` or `multiple` (only shown if more than one buffer is in use) | `never` |
 | `color-modes` | Whether to color the mode indicator with different colors depending on the mode itself | `false` |
+| `text-width` | Set the maximum line width for text wrapping with :reflow | `80` |
```

Comment:
```suggestion
| `text-width` | Set the maximum line width for text wrapping with `:reflow` | `80` |
```

Reply:
this should also contain a reference to soft-wrap now that #5420 is merged